homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -307,7 +307,8 @@ def __init__(
         self,
         config_entry: config_entries.ConfigEntry,
         options_flow: dict[str, SchemaFlowFormStep | SchemaFlowMenuStep],
-        async_options_flow_finished: Callable[[HomeAssistant, Mapping[str, Any]], None],
+        async_options_flow_finished: Callable[[HomeAssistant, Mapping[str, Any]], None]
+        | None = None,
```

Comment:
We could also default to `lambda _: None` but I think this is cleaner.

If we don't provide a default, then it would need to be explicit when initialising from integrations:
`return SchemaOptionsFlowHandler(config_entry, OPTIONS_FLOW, lambda _: None)`

Reply:
You could also let the default be `SchemaConfigFlowHandler.async_options_flow_finished`?

In any case, please also add a comment explaining the default value is there to make it easier to use `SchemaOptionsFlowHandler` in a config flow which is not a subclass of `SchemaConfigFlowHandler`

Reply:
I think default `None` is clearer. I have adjusted the comment.